src/crimson/net/Protocol.cc

```diff
@@ -95,6 +95,64 @@ void Protocol::notify_keepalive_ack()
   }
 }
 
+seastar::future<> Protocol::write_worker()
```

Comment:
`write_worker` IMHO suggests a bit that there is a separated entity (thread?) responsible for putting the bytes on wire. Maybe we could preserve the verb-based convention.